dominion/sql/allow_farmers_retain_land_ownership.sql

```diff
@@ -0,0 +1,124 @@
+--
```

Comment:
Lets start Dominion with just basic information during the development phase (since `our-land` has all this information already).